### PR TITLE
Add initial help file

### PR DESCRIPTION
**Pull request by BrandonRomano**
@hhsnopek adding my initial help file.  Your implementation wasn't printing the descriptions for some reason:

```sh
printf "\033[1;34mCommands:\033[1;0m\n"
cd "$(dirname "$0")"
for file in ./*; do
    if [[ -x "$file" ]]; then
      description=$(awk '/\#\#\s.*/{ print $0 }' ${file})
      printf "  \033[1;35m${file/\.\//}\033[1;0m\t\t${description/\#\#/}\n"
    fi
done
```

### CODE REVIEW

File: help [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+## Prints help dialog for all scripts in the directory of the help file
+cd "$(dirname "$0")"
+
+printf "\033[1;34mCommands:\033[1;0m\n"
+for file in ./*; do
+    if [[ -x "$file" ]]; then
+        filename="$(basename "$file")"
```
Comment (hhsnopek) [inline help]: do we want to create a subshell here or perform some string manipulation?

**Comment by hhsnopek** [comment]
here's my rendition:
```
#!/bin/bash

## Prints help dialog for all scripts in the directory of the help file
printf "\033[1;34mCommands:\033[1;0m\n"
cd "$(dirname "$0")"

for file in ./*; do
    if [[ -x "$file" ]]; then
      description="$(awk '/\#\#\s.*/{ print $0 }' ${file})"
      separator="$(printf "%-$((20 - ${#file}))s" ' ')"
      printf "  \033[1;35m${file/\.\//}\033[1;0m${separator}${description/\#\#/}\n"
    fi
done
```

**Comment by BrandonRomano** [comment]
Looks legit @hhsnopek I will plug that in

**Comment by BrandonRomano** [comment]
Oh wait, no this still isn't working on my computer (basically mac default programs)... Going to look into why

**Comment by BrandonRomano** [comment]
What version of awk are you on?  I am on this:

```
awk version 20070501
```

**Comment by BrandonRomano** [comment]
@hhsnopek, merging this guy in... I switched `\s` to `\ `, which I think should be more portable.

If you could run this on your comp and let me know that this still works that would be 👍 👍

**Pull request merged**